plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/editor/ERDEditorPart.java

```diff
@@ -1215,6 +1216,7 @@ public void run()
                     }
                 }
             }
+            refreshDiagram(true, true);
```

Comment:
As far as I remember, we intentionally don't refresh full diagram here as if the diagram is large it take noticeable amount of time. Isn't there another way of rebuilding relations without it?